src/webui/www/private/confirmdeletion.html

```diff
@@ -38,14 +85,28 @@
                 }).send();
             });
         });
+
+        function enableRememberBtn() {
+            let btn = $('rememberBtn');
+            btn.disabled = false;
+            btn.getElementsByTagName('path')[0].style.fill = "";
+        }
+
+        function disableRememberBtn() {
+            let btn = $('rememberBtn');
+            btn.disabled = true;
+            btn.getElementsByTagName('path')[0].style.fill = "lightgray";
```

Comment:
The webui now supports light theme and dark theme. Would `lightgray` work in both themes?

Reply:
Here's how `lightgray` looks in dark mode - the button looks enabled:
![2023-12-26T17:49:11,503650592-09:00](https://github.com/qbittorrent/qBittorrent/assets/6210075/6382970e-7af5-4367-8092-6d7166b5b0a1)

Setting the fill to the same color as --color-border-default (`#545454`) from `palette.css` looks better to my eye:
![2023-12-26T18:01:02,433400928-09:00](https://github.com/qbittorrent/qBittorrent/assets/6210075/19b4f718-c605-422f-a185-5db337fc4d22)

A new issue with the dark mode change is that the cursor doesn't reflect the disabled state of the button due to this rule: 
https://github.com/qbittorrent/qBittorrent/blob/8dcc734df562885131dbd0c2cc4ec31e24361559/src/webui/www/private/css/style.css#L30-L35

Should I write new CSS for a disabled button that sets `pointer: auto`?

Reply:
Feel free to address it in either another PR or additional commits in this PR.